core/src/impl/Kokkos_ViewTracker.hpp

```diff
@@ -70,7 +70,8 @@ struct ViewTracker {
 
   track_type m_tracker;
 
-  ViewTracker() = default;
+  KOKKOS_INLINE_FUNCTION
+  ViewTracker() : m_tracker() {}
```

Comment:
I think I'd slightly prefer:

```suggestion
  KOKKOS_INLINE_FUNCTION
  ViewTracker()
#ifndef KOKKOS_COMPILER_INTEL  
    = default;
#else
    // Work around a bug in Intel version X.Y.Z; see issue #XXXX 
    : m_tracker() {}
#endif     
```